pkgs/development/python-modules/pylint/1.9.nix

```diff
@@ -13,7 +13,8 @@ buildPythonPackage rec {
 
   checkInputs = [ pytest pytestrunner pyenchant ];
 
-  propagatedBuildInputs = [ astroid six isort mccabe configparser backports_functools_lru_cache singledispatch ];
+  propagatedBuildInputs = [ astroid six isort mccabe configparser backports_functools_lru_cache singledispatch ]
+    ++ lib.optional (pythonOlder "3") setuptools;
```

Comment:
no need for the optional because this version of pylint is only used for python2

Reply:
30369cf7db5a7231a94ca28ea32e18b0365e46ec